README.md

```diff
@@ -18,11 +18,13 @@ SPARK is currently **incomplete**.
 This script can be run as a Docker image or run locally.
 
 To run as a docker image, you may use the provided docker-compose.
+Before you can use the docker-compose, you need to create a volume called: ``p-data``
 
-If you would prefer to run the service locally, bear in mind you may need to run a local PostgreSQL Database for the fact module.
+If you would prefer to run the service locally, bare in mind you may need to run a local 
+PostgreSQL Database or the provided sql service via docker-compose for mecha's fact module.
 
-### Building via Docker
-We have provided a `docker-compose.yml` suitably configured for building and running Mecha
+###Building via Docker
+We have provided a `docker-compose.yml` suitably configured for building and running mechasqueak
```

Comment:
I would advise against making a one-line documentation change in this PR, as it is not within the scope of this PR
(this also appears to have caused a merge conflict)

Reply:
this may have occured due to the 3-way merge this had to undergo when I nuked the git history

Reply:
according to git blame, you were the last person to touch that paragraph

Reply:
according to the diff, **your** pr changes this paragraph at this line.

Reply:
yes, as my PR has changes from develop already merged into it, but: https://puu.sh/AWpCT/6362f40406.png